offchain-modules/bridge-ckb-sdk/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "bridge_ckb_sdk"
```

Comment:
The convention is use `-` in rust package name, and when we use it, it will be changed to `_` automatically by rust.

In case we'll publish this crate, the name is not specific enough, it's better to use `force-eth-sdk`.

Reply:
very confused about this. If i change to bridge-ckb-sdk, cargo build will report an error indicating that library target names cannot contain hyphens. So I had to use `_` instead of `-`.
```
Caused by:
		 library target names cannot contain hyphens: bridge-ckb-sdk
```